### DISCUSSION TITLE
第一次在gt发帖：请问可否支持曲奇网盘呢？

### DISCUSSION THREAD

**Discussion by dew556** [AlistGo/alist#1027] [Q&A]
如题。
请问能不能支持曲奇网盘等老牌稳定网盘呢，在支持国外trainbit网盘，这个支持外链且是5T空间，我两年的文件还在还没翻车也不用科学上网。
或后天能支持自定义添加新网盘，比如post json:登陆、列表、创建、删除等，就像一些php电影站那样自行正则采集资源站一样，这样扩展性好，且有些网盘站挂了也可以用其它的。
现在闪电盘已经挂了可以取消了，分秒帧基本不能用了也可以取消了。谷歌、微软、夸克这些需要服务器中转的流量有吃不消，所以可否在doc里说明告知哪些是中转的。
如上建议，仅供参考。如有打扰，请忽略。

**Comment by anwen-anyi** [comment]
https://github.com/Xhofe/alist/issues/840
